views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
```

Comment:
This isn't a zinger saying jargon for the sake of jargon, it's a fair description of what this is. The page was intended for developers that wanted to build "unstoppable" blockchain apps.

Now, we have more general people coming, and we want to better reflect that too: but then it should be something that also describes "home of decentralized apps", "use apps that you can trust", "browse a more decentralized internet". Not "welcome to our community"